sct.py

```diff
@@ -490,7 +490,6 @@ def isatty(self):  # pylint: disable=no-self-use
 @click.option('-c', '--config', multiple=True, type=click.Path(exists=True), help="Test config .yaml to use, can have multiple of those")
 @click.option('-l', '--logdir', help="Directory to use for logs")
 def run_test(argv, backend, config, logdir):
-
```

Comment:
Wow. I don’t think it’s a good idea to include `sct.py` in this PR# just to remove one empty line :)